src/cli/repl/commands/cfg.ts

```diff
@@ -28,7 +28,7 @@ export const controlflowCommand: ReplCommand = {
 export const controlflowStarCommand: ReplCommand = {
 	description:  'Get a mermaid url of the control-flow graph of R code, start with \'file://\' to indicate a file',
 	usageExample: ':controlflow',
-	aliases:      [ 'cfg*' ],
+	aliases:      [ 'cf*' ],
```

Comment:
Following the same comment as above:

```suggestion
	aliases:      [ 'cfg*', 'cf*' ],
```